src/js/flags/util.ts

```diff
@@ -0,0 +1,55 @@
+import { Flag } from "./model";
+
+import { itemFlags } from './items';
+import { worldFlags } from './world';
+import { monsterFlags } from './monsters';
+import { shopFlags } from './shops';
+import { hardModeFlags } from './hard-mode';
+import { tweakFlags } from './tweaks';
+import { routingFlags } from './routing';
+import { glitchFlags } from './glitches';
+import { glitchFixFlags } from './glitch-fixes';
+import { easyModeFlags } from './easy-mode';
+import { debugModeFlags } from './debug-mode';
+
+export const getFlagForName = (flag: string): Flag | undefined => {
```

Comment:
It's just a way of digging through the FlagSections to find a given Flag object. Most of the flagset class runs with based on the Flag's name, so I needed a way to trace back to the Flag itself since I moved the conflicts to the Flag interface (can I use Flag more in that sentence?).

Reply:
You can, but you shouldn't.

I guess what I was getting at is where it's used?  Who is interested in the Flag interfaces?  In general, most of the business logic wants to use simple boolean methods on FlagSet, so it seems pretty isolated to just FlagSet internals.